src/pages/comparison/comparison.tsx

```diff
@@ -1,5 +1,268 @@
-import {ConstructionInfo} from "../../components"
+import { useState } from "react";
+import DeckGL from "@deck.gl/react";
+import {
+  MapController,
+  FlyToInterpolator,
+  COORDINATE_SYSTEM,
+  MapView,
+  WebMercatorViewport,
+} from "@deck.gl/core";
+import { StaticMap } from "react-map-gl";
+import { Tileset3D } from "@loaders.gl/tiles";
+import { Tile3DLayer } from "@deck.gl/geo-layers";
+import styled from "styled-components";
+import { getCurrentLayoutProperty, useAppLayout } from "../../utils/layout";
+import { I3SLoader } from "@loaders.gl/i3s";
+import { getElevationByCentralTile } from "../../utils";
+import { INITIAL_MAP_STYLE } from "../../constants/map-styles";
+
+interface layoutProps {
+  layout: string;
+}
+
+const INITIAL_VIEW_STATE = {
+  longitude: -120,
+  latitude: 34,
+  height: 600,
+  width: 800,
+  pitch: 45,
+  maxPitch: 90,
+  bearing: 0,
+  minZoom: 2,
+  maxZoom: 30,
+  zoom: 14.5,
+  transitionDuration: 0,
+  transitionInterpolator: null,
+};
+
+const TRANSITION_DURAITON = 4000;
+
+const VIEW = new MapView({
+  id: "main",
+  controller: { inertia: true },
+  farZMultiplier: 2.02,
+});
+
+const Container = styled.div<layoutProps>`
+  display: flex;
+  flex-direction: ${getCurrentLayoutProperty({
+    default: "row",
+    tablet: "column",
+    mobile: "column",
+  })};
+  margin-top: 60px;
+  height: calc(100% - 60px);
+`;
+
+const DeckWrapper = styled.div<layoutProps>`
+  width: ${getCurrentLayoutProperty({
+    default: "50%",
+    tablet: "100%",
+    mobile: "100%",
+  })};
+  height: 100%;
+  position: relative;
+`;
+
+const Devider = styled.div<layoutProps>`
+  width: ${getCurrentLayoutProperty({
+    default: "14px",
+    tablet: "100%",
+    mobile: "100%",
+  })};
+
+  height: ${getCurrentLayoutProperty({
+    default: "100%",
+    tablet: "8px",
+    mobile: "8px",
+  })};
+
+  background-color: #232430;
```

Comment:
Shall we create a file for css constants?

Reply:
done